madmom/audio/cepstrogram.py

```diff
@@ -197,7 +220,7 @@ def __new__(cls, spectrogram, filterbank=MelFilterbank,
             spectrogram = Spectrogram(spectrogram, **kwargs)
 
         # instantiate a Filterbank if needed
-        if issubclass(filterbank, Filterbank):
+        if inspect.isclass(filterbank) and issubclass(filterbank, Filterbank):
```

Comment:
Just pass in a filterbank instance. Then, the first argument of issubclass won't be a class which provokes the following error:

`TypeError: issubclass() arg 1 must be a class`

For example, in my evaluation script I am doing it like that, which causes an error without my fix:
> filterbank = Filterbank(filterdata, pyfft.bin_frequencies)
> mfcc = MFCC(pyfft, filterbank=filterbank, num_bands=no_filters, fmin=0, fmax=fs / 2)